sdk/test-utils/recorder-new/package.json

```diff
@@ -85,7 +85,7 @@
     "@types/nise": "^1.4.0",
     "@types/node": "^12.0.0",
     "@types/mock-require": "~2.0.0",
-    "@types/mock-fs": "~4.10.0",
+    "@types/mock-fs": "~4.13.1",
```

Comment:
Do we know why we use ~ here instead of ^?

Reply:
Must have been an overlook.
Or did you have a reason for `~`, @sadasant?

Reply:
I don’t remember! 😄

Reply:
Can you put a PR to do caret (`^`) instead, @JonathanCrd?

Reply:
@HarshaNalluru Yes, I can :)

Reply:
I believe `rush add` command adds `~` instead of `^` by default if not provided the caret flag.
Probably, that's the reason for its presence.